source/docs/articles/sites/multidev.md

```diff
@@ -90,16 +97,30 @@ A branch with no environment associated with it can be deleted by going to Multi
 
 Branches can be deleted locally and the commit can be pushed to Pantheon, but this may have unintended consequences if an environment is associated with it; use the interface instead.​
 
+## Rename a Branch
+There is an 11-character limit for branch names. If you push a branch to Pantheon that exceeds the character limit, it cannot become a cloud development environment (CDE). The solution is to rename the branch. This is only recommended if you don't have any other users working on this branch, or if you have already coordinated with them.
+
+1. Rename the branch. Example: `git branch -m old-branch-name new-name`
+2. Change the remote branch. Because you can't directly rename the remote branch, you'll need to delete the remote branch with the old name and re-create it with a new name. Start by getting the remote name:  
+`git remote -v`  
```

Comment:
You don't need to delete/re-create the branch to rename it - you can run `git branch -m new-name` while you have the branch in question checked out locally and then `git push origin new-name` to push it remotely. The name change automatically populates in the Multidev branch overview and the Create Environment button becomes clickable. 

@bmackinney can you confirm?